### ISSUE TITLE
`dprint v0.11.1` CLI file arguments and matching are broken for Windows

### ISSUE THREAD

**Issue by rivy** [dprint/dprint#344]
For the Windows platform, the `dprint` CLI is not expanding wildcards nor correctly matching files or globs.

```text
C:\...\dprint>dprint --version
dprint 0.11.1

C:\...\dprint>dprint --verbose check scripts\build-homepage.js
[VERBOSE]: Creating directory: C:\Users\Roy\AppData\Local\dprint\cache
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\cache-manifest.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugin-cache-manifest.json
[VERBOSE]: Checking path exists: ./.dprintrc.json
[VERBOSE]: Reading file: ./.dprintrc.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-typescript\dprint-plugin-typescript-0.40.3.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-markdown\dprint-plugin-markdown-0.5.1.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-json\dprint-plugin-json-0.8.0.cached
[VERBOSE]: Checking path exists: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-prettier\0.2.1\dprint-plugin-prettier.exe
[VERBOSE]: Globbing: ["scripts\\build-homepage.js", "!website/playground/build", "!scripts/build-website", "!**/node_modules", "!**/dist", "!**/target", "!**/*-lock.json"]
[VERBOSE]: Finished globbing in 10ms
No files found to format with the specified plugins. You may want to try using `dprint output-file-paths` to see which files it's finding.

C:\...\dprint>dprint --verbose check scripts\*.js
[VERBOSE]: Creating directory: C:\Users\Roy\AppData\Local\dprint\cache
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\cache-manifest.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugin-cache-manifest.json
[VERBOSE]: Checking path exists: ./.dprintrc.json
[VERBOSE]: Reading file: ./.dprintrc.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-typescript\dprint-plugin-typescript-0.40.3.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-markdown\dprint-plugin-markdown-0.5.1.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-json\dprint-plugin-json-0.8.0.cached
[VERBOSE]: Checking path exists: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-prettier\0.2.1\dprint-plugin-prettier.exe
[VERBOSE]: Globbing: ["scripts\\*.js", "!website/playground/build", "!scripts/build-website", "!**/node_modules", "!**/dist", "!**/target", "!**/*-lock.json"]
[VERBOSE]: Finished globbing in 10ms
No files found to format with the specified plugins. You may want to try using `dprint output-file-paths` to see which files it's finding.

C:\...\dprint>dprint --verbose check "scripts\*.js"
[VERBOSE]: Creating directory: C:\Users\Roy\AppData\Local\dprint\cache
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\cache-manifest.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugin-cache-manifest.json
[VERBOSE]: Checking path exists: ./.dprintrc.json
[VERBOSE]: Reading file: ./.dprintrc.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-typescript\dprint-plugin-typescript-0.40.3.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-markdown\dprint-plugin-markdown-0.5.1.cached
[VERBOSE]: Checking path exists: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-prettier\0.2.1\dprint-plugin-prettier.exe
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-json\dprint-plugin-json-0.8.0.cached
[VERBOSE]: Globbing: ["scripts\\*.js", "!website/playground/build", "!scripts/build-website", "!**/node_modules", "!**/dist", "!**/target", "!**/*-lock.json"]
[VERBOSE]: Finished globbing in 13ms
No files found to format with the specified plugins. You may want to try using `dprint output-file-paths` to see which files it's finding.
```

**Comment by rivy** [comment]
I just posted PR #345 fixing this issue.

**Comment by dsherret** [comment]
These examples work with forward slashes, but not back slashes on Windows. So yeah, that's a bug.

```
> dprint output-file-paths scripts/*.js
V:\dprint\scripts\build-homepage.js
V:\dprint\scripts\build-plugin-schemas.js
> dprint output-file-paths "scripts/*.js"
V:\dprint\scripts\build-homepage.js
V:\dprint\scripts\build-plugin-schemas.js
```

**Comment by rivy** [comment]
Unfortunately, with the "revert", Windows paths are [still broken](https://github.com/dprint/dprint/pull/345#discussion_r593944391).

**Comment by rivy** [comment]
See <https://github.com/dprint/dprint/pull/345#issuecomment-798988633> for an alternative fix that leaves config alone.

**Comment by dsherret** [comment]
Should be fixed by 0d8ad763fbf6a5e8ecb15e8209d37f845f1545f4 and 8debade26d29f174f8009053d067cbad153daf70.
